tests/snapshot.cpp

```diff
@@ -44,8 +44,8 @@ BOOST_AUTO_TEST_CASE(snapshot)
 {
     brayns::SnapshotParams params;
     params.format = "png";
-    params.size = {50, 50};
```

Comment:
Because I wanted to reuse the image for the clipping plane tests. I can make all test images 50 x 50 as well. For my display 50 by 50 was too small, but for the test probably doesn't matter